### ISSUE TITLE
sgadmin - no output on early ClassNotFoundException

### ISSUE THREAD

**Issue by hackery** [floragunncom/search-guard#639]
* ES 6.5.4
* SG 6:6.5.4-24.1
* Oracle jre1.8.0_171
* Windows Server 2012

While leading one of our ops team through a standalone sgadmin install (on a machine where I don't have access) we encountered a problem at the outset where running sgadmin produced no output at all, and took no action. It turned out to be a badly-extracted ZIP file, which had left a partial jar collection in `.\lib` - but this failure behaviour is poor. I eventually asked them to remove the `2> NUL` ending from the invocation in `sgadmin.bat` and then we could see this:

```
D:\...\sgadmin\tools>sgadmin_test.bat -icl -nhnv -key kirk-key.pem -cert kirk.pem -cacert root-ca.pem -t config -f sg_config.local.yml -h $host
Error: A JNI error has occurred, please check your installation and try again
Exception in thread "main" java.lang.NoClassDefFoundError: org/elasticsearch/client/transport/NoNodeAvailableException
...
Caused by: java.lang.ClassNotFoundException: org.elasticsearch.client.transport.NoNodeAvailableException
```

I realise that there might be some verbose output that you want to discard, but failing during JVM startup is not a case that should be obscured. This wasted a disproportionate time in diagnosis, and helped create a mood of distrust of the product and process we're asking them to follow.

**Comment by floragunncom** [comment]
We will rethink this, thx for reporting.

Was the exit code of the script appropriate?

**Comment by hackery** [comment]
Re exit code - I'm afraid I don't have access to that environment, especially as we fixed up the lib/ directory to make it work already - I could try reproduce it here but I'm not sure that'd be conclusive.

**Comment by floragunncom** [comment]
Partially fixed with v24.2 released today. We will provide an additional fix with v25.0.

**Comment by floragunncom** [comment]
Fixed with v25.0 which was released today
